docs/prow/prow-installation-on-forks.md

```diff
@@ -69,7 +69,17 @@ For the purpose of the installation, you must have a set of service accounts and
  - **kyma-bot-npm-token** which is a token for publishing npm packages.
  - **sap-slack-bot-token** which is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).
  - **sa-kyma-artifacts** service account with the role that allows the account to store objects in a bucket.
-    
+
+3. Create a Secret for the workload cluster in the Prow Cluster:
+
+    ```bash
+        export PROJECT={project-name}
+        export CLUSTER_NAME={cluster-name} ###  Use a different name as that of the Prow cluster
+        export ZONE={zone-name}
```

Comment:
or maybe env variable is out-dated, in the script I can find: WORKLOAD_CLUSTER_NAME. Also in the script there is another required env, GOPATH which is missing here in the documentation.

Reply:
Added